deployments/smoke/smoke.tf

```diff
@@ -220,3 +225,18 @@ output "guest_password" {
   value     = random_string.guest_password.result
   sensitive = true
 }
+
+locals {
+  # used to omit configuration options if the concourse version is too low
+  # semver x.y.z
+  left = regex("(\\d)\\.(\\d)\\.(\\d).*", var.concourse_version)
+  x1   = local.left[0]
+  y1   = local.left[1]
+  z1   = local.left[2]
+
+  x2 = 6
+  y2 = 5
+  z2 = 0
+
+  gt_650 = (local.x1 > local.x2) || (local.x1 == local.x2 && local.y1 > local.y2) || (local.x1 == local.x2 && local.y1 == local.y2 && local.z1 > local.z2)
```

Comment:
This should be `gte`, right? (as should the corresponding conditions, not just the name)

Reply:
yeah from the (crossed out) pr and commit message(s) looks like it should be 6.5 and above (gte) using CONCOURSE_RUNTIME